plugin/mouse/continuous_scrolling.talon

```diff
@@ -0,0 +1,3 @@
+tag: user.continuous_scrolling
```

Comment:
I think we need to ensure this coexists well with the tag user.numbers_unprefixed. Since the talon files have the same specificity, it's ambiguous which one talon will choose. Probably just add `not tag: user.continuous_scrolling` to that file

Reply:
specificity doesn't work on commands right now besides exact matches. it's pretty complicated to factor in .talon specificity in the command parser.

Reply:
In this case, the commands are not an exact match. unprefixed numbers use a different capture, so I don't think an action really nets us anything in this context either.